app/code/core/Mage/Sales/Model/Order/Creditmemo.php

```diff
@@ -224,6 +224,25 @@ protected function _initOldFieldsMap()
         return $this;
     }
 
+    /**
+     * Load creditmemo by increment id
+     *
+     * @param string $incrementId
+     * @return $this
+     */
+    public function loadByIncrementId($incrementId)
+    {
+        $ids = $this->getCollection()
+            ->addAttributeToFilter('increment_id', $incrementId)
+            ->getAllIds();
+
+        if (!empty($ids)) {
+            reset($ids);
+            $this->load(current($ids));
+        }
```

Comment:
```suggestion
        return $this->load($incrementId, 'increment_id');
```

Reply:
I copied/pasted _Mage_Sales_Model_Order_Invoice_, but why not!